method/README.md

```diff
@@ -29,12 +29,9 @@ Here are the most relevant categories identified:
 - Basic office furniture
 - Meals
 
-Due to the expected use case, the CO2 equivalent data in the database corresponds to a **cradle to gate** evaluation (i.e. Impact of resource extraction + Manufacturing). The use phase corresponds mostly to energy use and is thus included in the company energy consumption.
-
+CO2 equivalent data of services such as energy and transport include the infrastructures’ life cycles (i.e. power plants, roads, cars). Due to the expected use case, the CO2 equivalent data of products (IT equipment and office furniture) in the database corresponds to a **cradle to gate** evaluation (i.e. Impact of resource extraction + Manufacturing). The use phase of purchased products (i.e. IT) corresponds mostly to energy consumption and is thus included in the operational stage of the company.
```

Comment:
- include the infrastructure (no "'") life cycle (no "s")
- cradle to gate -> cradle-to-gate